tests/components/stream/common.py

```diff
@@ -0,0 +1,62 @@
+"""Collection of test helpers."""
+import io
+
+from homeassistant.components.stream import request_stream, get_stream
+from homeassistant.loader import bind_hass
+
+
+def generate_h264_video():
+    """
+    Generate a test video.
+
+    See: http://docs.mikeboers.com/pyav/develop/cookbook/numpy.html
+    """
+    import numpy as np
```

Comment:
Would it make more sense to:
 - Just generate a black video?
 - Store a test video in the test folder?

Reply:
Potentially.  My mindset was stuck in "don't read/write anything from/to disk".  Generating the video is pretty quick though.  I think in initial testing it took just as long to open the video from disk as it did to create and load from memory.  I will create a simple script to test timing of both.

Reply:
Sure, but now our tests rely on numpy. Which I guess already got pulled in from another integration?

Reply:
Yeah, `numpy` was already a test dep from a separate integration.